pages/about.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const About = () => {
```

Comment:
Oh good point I didn't put them in the nav but these pages are reachable from the buttons on the home page

Reply:
Oh mb I missed those, they kind of blended in. Might want to either just have the buttons with the shadow effect instead of the cards and images, or you could have them distinguished by colour? Since they all look like buttons